Bottom: 48b41b87a3c9ee710fe3d58ed6b42640d0d8dcce
Top:    c0b14a482ec46ca2db76509af2590a56d9825333
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-06-03 23:35:57 +0100

Batch size for accumulating lines down to 1

This works around a bug seen on one of my boards, and is similar
to what I had in the non-pours variant of the code.


---

diff --git a/src/pour.c b/src/pour.c
index 501f703..b4b1686 100644
--- a/src/pour.c
+++ b/src/pour.c
@@ -68,7 +68,7 @@ RCSID ("$Id$");
 
 #define UNSUBTRACT_BLOAT 100
 #define SUBTRACT_PIN_VIA_BATCH_SIZE 100
-#define SUBTRACT_LINE_BATCH_SIZE 20
+#define SUBTRACT_LINE_BATCH_SIZE 1
 
 /* ---------------------------------------------------------------------------
  * local prototypes
